#ifndef mnlc_simple_costmap_H
#define mnlc_simple_costmap_H
#include <geometry_msgs/PointStamped.h>
#include <nav_msgs/OccupancyGrid.h>
#include <tf/transform_listener.h>
#include <nav_msgs/GetMap.h>
#include <std_srvs/Empty.h>
#include "ros/ros.h"
#include <map>

nav_msgs::OccupancyGrid mapdata;
#endif